src/client/dfuse/il/int_posix.c

```diff
@@ -1404,6 +1409,62 @@ dfuse_fclose(FILE *stream)
 	return __real_fclose(stream);
 }
 
+DFUSE_PUBLIC int
+dfuse___fxstat(int ver, int fd, struct stat *buf)
+{
+	struct fd_entry *entry = NULL;
+	int rc;
+
+	rc = vector_get(&fd_table, fd, &entry);
+	if (rc != 0)
+		goto do_real_fstat;
+
+	/* Turn of this feature if the kernel is doing metadata caching, in this case it's btter
```

Comment:
Only if you need to push again
```suggestion
	/* Turn off this feature if the kernel is doing metadata caching, in this case it's btter
```

Reply:
I got so confused with caching vs not caching and when the feature was on or off when writing this PR, maybe this is why.  I think we probably *don't* want this in TB3 so I'll likely need to re-push and will fix this then.